bedrock/products/templates/products/vpn/includes/macros.html

```diff
@@ -53,6 +53,14 @@
   </div>
 {% endmacro %}
 
+{% macro vpn_nav_platform_cta(alt_link_text) -%}
+  <div class="vpn-nav-cta">
+    <a class="mzp-c-button mzp-t-secondary mzp-t-md hide-from-legacy-ie" href="{{ url('products.vpn.landing') }}" data-cta-type="button" data-cta-text="Join the VPN Waitlist" data-cta-position="navigation">
```

Comment:
This CTA isn't reliant on JS, so you can remove the `hide-from-legacy-ie` class name?

Suggest also chanigng `data-cta-text="Join the VPN Waitlist"` to something else for GA.

Reply:
Should the `data-cta-text` be unique for GA, or just reflect the text of the button? (It now says "See pricing & availability")

Reply:
The value should be something descriptive that's the same for all locales (in English), so "See pricing & availability" works 👍

Reply:
I removed all data-attributes from the link, as its internal/doesn't need even tracking.